src/interfaces/modular/Minimizer_includes.i

```diff
@@ -0,0 +1,5 @@
+%{
+ #include <shogun/optimization/Minimizer.h>
+ #include <shogun/optimization/FirstOrderMinimizer.h>
+ #include <shogun/optimization/lbfgs/LBFGSMinimizer.h>
```

Comment:
@karlnapf 
If I want to add nlopt in the i file, how can I guard nlopt when `nlopt` is missing/disabled?